src/components/details/MatchDetails.jsx

```diff
@@ -55,14 +58,20 @@ export default function MatchDetails({ open, handleClose, matchId, title, reserv
       />
       <BootstrapDialog
         onClose={handleClose}
+        maxWidth='xl'
         aria-labelledby="customized-dialog-title"
         open={open}
       >
         <BootstrapDialogTitle style={{ color: 'white' }} onClose={handleClose}>
           {label(title)}
         </BootstrapDialogTitle>
-        <DialogContent dividers>
-          {matchDetails && <MatchDetailsContent matchDetails={matchDetails} />}
+        <DialogContent className={classes.mainContainer} >
+          <Grid className={classes.cardContent}>
+            {matchDetails && <MatchDetailsContent matchDetails={matchDetails} />}
+            <Grid item xs={4} sx={{ marginTop: '2vh' }}>
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/FeNixCrew/MiEntradita-frontend/pullRequest?prid=8421134)
```suggestion
            <Grid item xs={4} sx={{ marginTop: "2vh" }}>
```